vsix/check_installed.ps1

```diff
@@ -16,7 +16,7 @@ $ntimes = 5
 For ($i=1; $i -le $ntimes; $i++) {  # Run 10 times
     $output = . "$devenv" /ConanVisualStudioVersion /?
 
-    $pattern = "^${env:APPVEYOR_BUILD_VERSION}\s+Microsoft Visual Studio"  # Version + output from /? command
+    $pattern = "^${env:APPVEYOR_BUILD_VERSION}\s+"  # Version + output from /? command
```

Comment:
Is this change still needed?

Reply:
yes, because of extra output from devenv.exe (warnings)

Reply:
anyway, it's not a big deal, as we're going to verify by actual integration test rather than by just version check